src/app/components/CommentCard/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { View, Image, Text } from 'react-native';
+import { Comment } from '@interfaces/comment';
+
+import styles from './styles';
+
+interface Props extends Comment {}
+
+function CommentCard({ avatar, author, content }: Props) {
+  return (
+    <View style={styles.comment}>
+      <Image source={avatar} style={styles.avatar} />
```

Comment:
Missing `resizeMode` for the Image

Reply:
Default value for `resizeMode` is `cover`. That's is what I'm looking for.